styles.css

```diff
@@ -0,0 +1,93 @@
+form{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  margin: 4% auto;
```

Comment:
try to avoid using margin as percent (except 0 and 100) it may cause issue in different screens